check/check.go

```diff
@@ -68,6 +72,10 @@ type Result struct {
 	Signature string `json:"signature"`
 }
 
+func init() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
As @oxtoacart suggested before in another PR, instead of changing the default Source which affects all `rand` calls within same process (that means any application which uses this library), we can call `rand.New(rand.NewSource(time.Now().UnixNano())` to define a package level `Rand` object. Not a deal though.